packages/react-native/sdks/hermes-engine/hermes-utils.rb

```diff
@@ -5,6 +5,7 @@
 
 require 'net/http'
 require 'rexml/document'
+require 'open3'
```

Comment:
Can you also add a diff tag to the end of this line?
```suggestion
require 'open3' # [macOS]
```

Reply:
Opps, fixed